render/expected/expected.go

```diff
@@ -16,566 +14,110 @@ var (
-			Shape: circle,
-			Node:  report.MakeNode().WithAdjacent(ServerEndpointID),
-			EdgeMetadata: report.EdgeMetadata{
-				EgressPacketCount: newu64(50),
-				EgressByteCount:   newu64(500),
-			},
-		},
-		RandomClientEndpointID: {
-			ID:    RandomClientEndpointID,
-			Shape: circle,
-			Node:  report.MakeNode().WithAdjacent(ServerEndpointID),
-			EdgeMetadata: report.EdgeMetadata{
-				EgressPacketCount: newu64(60),
-				EgressByteCount:   newu64(600),
-			},
-		},
-		NonContainerEndpointID: {
-			ID:           NonContainerEndpointID,
-			Shape:        circle,
-			Node:         report.MakeNode().WithAdjacent(GoogleEndpointID),
-			EdgeMetadata: report.EdgeMetadata{},
-		},
-		GoogleEndpointID: {
-			ID:    GoogleEndpointID,
-			Shape: circle,
-			Node:  report.MakeNode(),
-		},
-	}).Prune()
-
-	ClientProcess1ID      = render.MakeProcessID(fixture.ClientHostID, fixture.Client1PID)
-	ClientProcess2ID      = render.MakeProcessID(fixture.ClientHostID, fixture.Client2PID)
-	ServerProcessID       = render.MakeProcessID(fixture.ServerHostID, fixture.ServerPID)
-	nonContainerProcessID = render.MakeProcessID(fixture.ServerHostID, fixture.NonContainerPID)
-	unknownPseudoNode1ID  = render.MakePseudoNodeID(fixture.UnknownClient1IP)
-	unknownPseudoNode2ID  = render.MakePseudoNodeID(fixture.UnknownClient3IP)
-
-	unknownPseudoNode1 = func(adjacent string) render.RenderableNode {
-		return render.RenderableNode{
-			ID:     unknownPseudoNode1ID,
-			Label:  fixture.UnknownClient1IP,
-			Pseudo: true,
-			Shape:  circle,
-			Node:   report.MakeNode().WithAdjacent(adjacent),
-			Children: render.MakeRenderableNodeSet(
-				RenderedEndpoints[UnknownClient1EndpointID],
-				RenderedEndpoints[UnknownClient2EndpointID],
-			),
-			EdgeMetadata: report.EdgeMetadata{
-				EgressPacketCount: newu64(70),
-				EgressByteCount:   newu64(700),
-			},
-		}
-	}
-	unknownPseudoNode2 = func(adjacent string) render.RenderableNode {
-		return render.RenderableNode{
-			ID:     unknownPseudoNode2ID,
-			Label:  fixture.UnknownClient3IP,
-			Pseudo: true,
-			Shape:  circle,
-			Node:   report.MakeNode().WithAdjacent(adjacent),
-			Children: render.MakeRenderableNodeSet(
-				RenderedEndpoints[UnknownClient3EndpointID],
-			),
-			EdgeMetadata: report.EdgeMetadata{
-				EgressPacketCount: newu64(50),
-				EgressByteCount:   newu64(500),
-			},
-		}
-	}
-	theIncomingInternetNode = func(adjacent string) render.RenderableNode {
-		return render.RenderableNode{
-			ID:         render.IncomingInternetID,
-			Label:      render.InboundMajor,
-			LabelMinor: render.InboundMinor,
-			Pseudo:     true,
-			Shape:      cloud,
-			Node:       report.MakeNode().WithAdjacent(adjacent),
-			Children: render.MakeRenderableNodeSet(
-				RenderedEndpoints[RandomClientEndpointID],
-			),
-			EdgeMetadata: report.EdgeMetadata{
-				EgressPacketCount: newu64(60),
-				EgressByteCount:   newu64(600),
-			},
+	// Helper to make a report.node with some common options
```

Comment:
Like the new helper function!

But we seem to have lost all the children testing.  I think its important we keep that.